actionpack/lib/action_controller/metal/live.rb

```diff
@@ -163,6 +163,11 @@ def write(string)
         end
       end
 
+      # Same as +write+ but automatically include a newline at the end of the string.
+      def writeln(string)
+        write "#{string}\n"
```

Comment:
Do we want to guard for the line already ending in a `\n`? This was just lifted from my comparison to `IO#puts`, which does that.

Reply:
Yeah I dig that.